### PR TITLE
Update bashio 0.12.1

### PR DESCRIPTION
**Pull request by pvizeli**

### CODE REVIEW

**Comment by frenck** [comment]
I have not tested this myself yet... Have you?

**Comment by pvizeli** [comment]
tested and work

**Comment by frenck** [comment]
👍 

I've pushed v0.12.1 with your suggestion. Not a real requirement for this PR, but... 🤷

File: alpine/aarch64/Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ ENV \
 SHELL ["/bin/ash", "-o", "pipefail", "-c"]
 
 # Version
-ARG BASHIO_VERSION=0.11.0
+ARG BASHIO_VERSION=0.12.0
```
Comment (frenck) [inline alpine/aarch64/Dockerfile:17]: Now available with the `/tmp/.bashio` tweak

```suggestion
ARG BASHIO_VERSION=0.12.1
```

**Review by frenck — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**